.github/workflows/flutter-action.yml

```diff
@@ -0,0 +1,32 @@
+name: Flutter Action
+
+on:
+  push:
+    paths-ignore:
```

Comment:
Am not sure if the list format (i.e. `[push, pull-request] ) works for this, I haven't seen an example like it yet.

Reply:
Ignore this as we probably don't need to double up